src/components/auth/RegisterForm.js

```diff
@@ -79,6 +79,9 @@ const RegisterForm = ({ initialValues: _iv, lockReferralCode, enterprise }) => {
     >
       {({ isSubmitting, errors }) => (
         <Form style={{ flex: 1, width: "100%" }}>
+          <div className=" d-flex justify-content-center">
+            {logo && (<div className="mb-5 mt-2"><img src={`${process.env.REACT_APP_API_URL}/Logos/${logo}`} alt="logo" width="50" height="50" /></div>)}
```

Comment:
If we set the size with `max-width` and `max-height` it shouldn't stretch the logo to fit. I also increased the width to avoid the logo being really small if it's wide and there is plenty of horizontal space given it's on it's own line.
```suggestion
            {logo && (<div className="mb-5 mt-2"><img src={`${process.env.REACT_APP_API_URL}/Logos/${logo}`} alt="logo" max-width="250" max-height="50" /></div>)}
```

Reply:
change to 
.logo-image {    max-width: 20%;    height: auto;}